docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -37,11 +38,9 @@ export const mdComponents: MDComponents = {
       {children.map(child => (child?.props?.node?.tagName === 'p' ? child?.props.children : child))}
     </Quote>
   ),
-  code: ({ children, node }) =>
-    Array.isArray(node.properties?.className) ? (
-      <Code className={node.properties?.className[0].toString() || 'language-unknown'}>
-        {children}
-      </Code>
+  code: ({ children, className }) =>
+    className ? (
+      <Code className={className || 'language-unknown'}>{children}</Code>
```

Comment:
If className is truthy here, the prop's value expression will never evaluate to `language-unknown`

Reply:
Yup, it is leftover from previous implementation, I clean this up! 🙂